docs/dashboards/edit-mode.md

```diff
@@ -0,0 +1,49 @@
+Edit mode
+===
+
+Edit mode is a mode, in which the user can change the appearance of the dashboard through the UI. To turn on this feature, you need to add the class to main dashboard div:
+```html
+    <div id="container" class="highcharts-dashboard"></div>
+```
+
+
+Also in the dashboard config, this option needs to be enabled:
+```javascript
+    editMode: {
+            enabled: true,
+            contextMenu: {
+                enabled: true,
+                items: ['editMode']
+            }
+        },
+```
+
+Then, the edit mode option can be enabled by the context menu in the upper right corner.
+
+![edit-mode-context-menu.png](edit-mode-context-menu.png)
+
+When edit mode is enabled, the appearance of the dashboard changes. When a cell is hovered it gets blue border and the row, in which the cell is, gets green border. Additionally both row and cell get a tooltip, which allows the user to change its position, change its options, or just delete it.
```

Comment:
`row, in which the cell is` => `row, containing the cell,` or at least `row, in which the cell is present`